src/lib/utils/sendBeacon.ts

```diff
@@ -7,26 +7,22 @@
  */
 import * as debug from "debug";
 
-const http = require("http-client");
-
 import "isomorphic-fetch";
 
 import { IManifest } from "../types";
 
 const dbg: debug.IDebugger = debug("@stylelounge/http-queue:utils:sendBeacon");
 
 const sendHttp = async (manifest: IManifest) => {
-    const {createFetch, accept, json, method} = http;
-
-    const post = createFetch(
-        method(manifest.verb.toUpperCase()),
-        accept("application/json"),
-        json(manifest.data || {})
-    );
-
     dbg(`Sending data via "XHR".`);
-
-    await post(manifest.url);
+    await window.fetch(new Request(manifest.url), {
```

Comment:
http-queue uses `isomorphic-fetch` to work in the browser and node.js (https://github.com/matthew-andrews/isomorphic-fetch) In the latter there's no `window`object so this would break compatibility with node.js